File: 169-majority-element/majority-element.go
func majorityElement(nums []int) int {

	var maps = make(map[int]int)

	for _, num := range nums {
		maps[num] = maps[num] + 1
	}

	max := 0
	val := 0
	for index, freq := range maps {
		if max < freq {
			max = freq
			val = index
		}
	}
	return val

}

File: 169-majority-element/majority-element.py
class Solution(object):
    def majorityElement(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        min = len(nums)//2
        dict_ = {}
        for i in nums:
            dict_[i] = 1+dict_.get(i,0)
            if dict_[i] >min:
                return i